README.md

```diff
@@ -317,62 +317,39 @@ Tips:
 
 # Development
 
-### Project setup
-
 - Language
-  - Java jdk-9.0.0 or higher
-  - and Kotlin (latest version) are both required.
+  - Java jdk-9
+  - Kotlin (latest version)
 - IDE: due to use of Kotlin, Intellij Idea is strongly recommended, although not required
-- dependencies
-  - Java/Kotlin: all jars are included in /working dir/lib
-  - VLC: installation must be placed in working dir/vlc directory. Obtain latest [here](https://www.videolan.org/vlc/). Without this step, audio playback will not be possible.
-  - Kotlin compiler: kotlinc must be placed in working dir/kotlinc directory. Obtain latest [here](https://github.com/JetBrains/kotlin/releases). Without this step, compiling widgets written in Kotlin will not be possible.
```

Comment:
Some of these are still important information, I will add new section and include it.

Reply:
These now moved to the "Preparations" section

Reply:
I wanted to make an "Project structure" section that would explain what it is the dev is looking at once the build succeeds.